src/Inventory/Asset/InventoryAsset.php

```diff
@@ -311,12 +311,14 @@ public function getAgent(): Agent
      * Set entity id from main asset
      *
      * @param integer $id Entity ID
+     * @param integer $is_recursive
      *
      * @return $this
      */
-    public function setEntityID($id): InventoryAsset
+    public function setEntityID($id, $is_recursive = 0): InventoryAsset
```

Comment:
Maybe add a new new `setEntityRecursive` method or rename to `setEntity` (and change phpdoc).